low-level/sensors/impl/generic/node_memory_fault.py

```diff
@@ -111,12 +111,9 @@ def initialize(self, conf_reader, msgQlist, product):
             self._utility_instance = self._utility_instance or \
                                 self.tool_factory.get_instance(mem_fault_utility)
 #            self._utility_instance.initialize()
-        except KeyError as key_error:
-            logger.error(
-                "Unable to get the instance of {} \
-                Utility. Hence shutting down the sensor {}"\
-                .format(mem_fault_utility, MemFaultSensor.SENSOR_NAME))
-            self.shutdown()
+        except Exception as err:
+            raise Exception("Error while initializing MemFaultSensor. "
+                f"Unable to get the instance of {mem_fault_utility} utility, {err}")
```

Comment:
`raise with e` as described in the previous file.

Reply:
As justified above.